src/test/java/spoon/test/annotation/AnnotationTest.java

```diff
@@ -544,6 +544,7 @@ public void testUsageOfTypeAnnotationBeforeExceptionInSignatureOfMethod() throws
 	@Test
 	public void testUsageOfTypeAnnotationInReturnTypeInMethod() throws Exception {
 		final Launcher launcher = new Launcher();
+		launcher.getEnvironment().setNoClasspath(false);
```

Comment:
why do we need those changes? if it works in full classpath, it should work in noclasspath, correct?

Reply:
Actually it works but the printed result is not exactly the same, so instead of fixing the assert I decided to keep the old setup.

Here the obtained diff if we call the test with a noclasspath:
```
Return type with an type annotation must be well printed expected:
<[public java.lang.@spoon.test.annotation.testclasses.TypeAnnotation
]String m3() {
    re...> but was:<[@spoon.test.annotation.testclasses.TypeAnnotation
public java.lang.]String m3() {
    re...>
```

Reply:
does this mean that we don't have the fully-qualified names in noclasspath or not?

Reply:
No it means that in noclasspath, the annotation seems not printed exactly at the same place: before or after the modifier of a field.